src/System.Diagnostics.Process/src/PinvokeAnalyzerExceptionList.analyzerdata.uap

```diff
@@ -2,4 +2,3 @@ advapi32.dll!OpenProcessToken
 kernel32.dll!GetProcessAffinityMask
 kernel32.dll!SetProcessAffinityMask
 kernel32.dll!SetThreadAffinityMask
-ntdll.dll!NtQuerySystemInformation
```

Comment:
Why would you need to add this if this file is only used in uap? This is wrong, shouldn't be here because in theory you are removing the usage of this API in uap.